index.js

```diff
@@ -1,27 +1,42 @@
 const express = require("express");
 const mongoose = require("mongoose");
 const cors = require("cors");
-const router = require("./routes/index");
-const { DATABASE_URL } = require("./config");
+const router = require("./routes");
+const jwt = require("jsonwebtoken");
+const { DATABASE_URL, JWT_SECRET_KEY } = require("./config");
+const fileUpload = require("express-fileupload");
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
-app.use("/", router);
+app.use(express.static("avatars"));
+app.use(fileUpload({}));
+app.use("/api", router);
 app.use(
   cors({
     origin: "*",
     credentials: true,
   })
 );
 app.use((request, response, next) => {
-  if (request.headers.authorization) {
-    const token = request.headers.authorization;
-    request.is_authorized_user = token === TOKEN;
+  try {
```

Comment:
I'd like to see this function at /middleware/auth.js

Reply:
done + fixed issue below (authorization header)